recipes/squash/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -ex
+
+EXTRA_CMAKE_ARGS=""
+if [[ `uname` == 'Darwin' ]]; then
+  export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+  # We override most toolchain variables to modify the min version for OSX.
+  export MACOSX_VERSION_MIN="10.7"
```

Comment:
Those are holdovers that need to be updated to use the `toolchain`. As is documented in issue ( https://github.com/conda-forge/toolchain-feedstock/issues/7 ), we actually can't effectively set the minimum version to 10.7. The best we can do is use 10.9 as it is the oldest SDK the XCode on Travis CI comes with.